reference/docs-conceptual/whats-new/Migrating-from-Windows-PowerShell-51-to-PowerShell-7.md

```diff
@@ -0,0 +1,257 @@
+---
+title: Migrating from Windows PowerShell 5.1 to PowerShell 7
+description: Migrate to PowerShell 7 with these helpful notes
+ms.date: 03/25/2020
+---
+
+# Migrating from Windows PowerShell 5.1 to PowerShell 7
+
+PowerShell 7 is a cross-platform (Windows, Linux, and macOS) automation tool and configuration
+framework optimized for dealing with structured data, REST APIs, and object models. PowerShell
+includes a command-line shell, object-oriented scripting language, and a set of tools for executing
+scripts/cmdlets and managing modules. Designed for Cloud, Hybrid-Cloud and on-premises, the latest
+release of PowerShell is packed with performance enhancements and features over previous versions.
+
+Migrate with minimal investment time and lower the risk of failure. PowerShell 7 is designed to work
+fully side-by-side with Windows PowerShell letting you easily and quickly test and compare before
+deployment.
+
+## PowerShell 7 Supported on Windows
+
+PowerShell 7 is ready to deploy for Microsoft Windows client and server operating systems.
```

Comment:
I wonder if we should highlight that a reboot is not needed unlike WMF which was a major pain point.